app/models/mixins/cloud_tenancy_mixin.rb

```diff
@@ -18,8 +13,8 @@ def tenant_id_clause_format(tenant_ids)
     end
 
     def tenant_joins_clause(scope)
-      scope.includes(QUERY_REFERENCES)
-           .references(QUERY_REFERENCES) # needed for the where to work
+      scope.includes(:cloud_tenant => "source_tenant", :ext_management_system => {})
+           .references(:cloud_tenant, :tenants, :ext_management_systems) # needed for the where to work
```

Comment:
should this be `.references(:cloud_tenants, ...)`  since it's the table name?

Reply:
yea, this is a good example of why we don't want the framework to just ignore bad refefences